package.json

```diff
@@ -13,6 +13,7 @@
     "vscode": "./ci/dev/vscode.sh",
     "vscode:patch": "./ci/dev/patch-vscode.sh",
     "vscode:diff": "./ci/dev/diff-vscode.sh",
+    "vscode:clean": "./ci/dev/clean-vscode.sh",
```

Comment:
Let's call this `reset`. `clean` would delete all build artifacts and caches which this doesn't do. it just resets the source.

Reply:
ah gotcha! good point. I'll update now.